### DISCUSSION TITLE
Release Notes CovPass Version 1.25

### DISCUSSION THREAD

**Discussion by alexcimander** [Digitaler-Impfnachweis/covpass-ios#186] [Announcements]
# Release Notes CovPass Version 1.25

Release 1.25 is currently in final review and is expected to be released within the next couple of hours. 
  
### **Main Features CovPass:**  💉 

- Redesign of the startscreen: On the start screen you can see again the type of certificate including the date of vaccination. A data-saving representation of the QR code can still be found in the details of the respective certificate.

### **Main Features CovPassCheck:**  💉 

- Combination recovery/basic immunization: A visual guide indicates to verifiers that recoveries older than 90 days can be valid in combination with basic immunizations.

 
-----------------------------------------------------------------------------------------------------------

An outlook for upcoming topics will be published soon!

-----------------------------------------------------------------------------------------------------------

**Thank you all for you support and feedback! Stay safe everyone** 👍 🎊 

Team CovPass and CovPassCheck! 🧑‍⚕️ 👩‍⚕️

**Comment by Ein-Tim** [comment]
@alexcimander

So RKI stepped back from the requirement to not show the type of certificate at the start screen?

**Reply by Ein-Tim** [reply]
And version 1.32 is now back to not showing the type of certificate on the start screen.
